mods/ts/sequences/infantry.yaml

```diff
@@ -140,6 +140,7 @@ oxanna:
 slav:
 	Inherits: ^BasicInfantry
 	Inherits@attack: ^BasicInfantryAttack
+	icon: xxicon
```

Comment:
Adding these *only* to this map-specific units to hide your yaml errors is horrible.  Please either add these for all the civilian actors, or even better work out why this fails and fix that.  Please also file a bug with a repro case so that this isn't forgotten.